src/mainField/MainField.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import style from "../App.module.css";
+
+import { StateIndicator } from "../indicators/StateIndicator";
+import { FoodIndicator } from "../indicators/FoodIndicator";
+import { AgeIndicator } from "../indicators/AgeIndicator";
+import ButtonFeed from "./ButtonFeed";
+
+export const MainField = ({ state, indicators }) => {
+  const [foodPercent, setFoodPercent] = useState(state.initialFoodPercent);
+  const [age, setAge] = useState(state.initialAge);
+  const [buttonFeedEntered, setButtonFeedEntered] = useState(false);
```

Comment:
maybe let's call it `buttonFeedEnabled`?

Reply:
renamed